daemon/endpoint.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/cilium/cilium/pkg/completion"
 	"github.com/cilium/cilium/pkg/controller"
 	"github.com/cilium/cilium/pkg/endpoint"
+	endpointid "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the previous commit in order not to break git bisecting as the previous commit alone will not compile.